apps/web/app/notFoundClient.tsx

```diff
@@ -82,7 +82,7 @@ export function NotFound({ host }: { host: string }) {
    **/
   if (isInsights) {
     return (
-      <div className="min-h-screen bg-white px-4" data-testid="404-page">
+      <div className="min-h-screen bg-white px-4" data-testid="app-router-not-found-page">
```

Comment:
This is being applied only for insights.

We need to add on line 107 as well or reuse the existing data-testid that is there